Assets/Scripts/EditorState/AvailableComponentsState.cs

```diff
@@ -31,41 +39,66 @@ public struct AvailableComponent
 
         private readonly IReadOnlyList<AvailableComponent> buildInComponents = new List<AvailableComponent>
         {
+            new AvailableComponent
+            {
+                category = "Build in",
```

Comment:
I think the correct spelling is "Builtin"

Reply:
Yes, but with a space or hyphen.
"Built in" or "Built-in"